src/JK/CommonBundle/Controller/AbstractController.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace JK\CommonBundle\Controller;
+
+class AbstractController
+{
+    
```

Comment:
Please implement some useful methods for your controller from this list, for example `render`
http://symfony.com/doc/current/cookbook/controller/service.html#base-controller-methods-and-their-service-replacements

You can find example of implementation in `Symfony\Bundle\FrameworkBundle\Controller\Controller`